src/core/task.ts

```diff
@@ -49,30 +48,29 @@ function run(): void {
 }
 
 export function shouldYield(method: Metric): boolean {
-    let elapsed = performance.now() - tracker[method];
-    return (elapsed > config.longtask);
+    let elapsed = performance.now() - tracker[method].start;
```

Comment:
Wouldn't this include time that was not processing? Not sure if your intent is to measure the "real-time" elapsed or the time the task was processing.